src/utils/openmctLocation.js

```diff
@@ -29,55 +29,47 @@ import objectUtils from '../api/objects/object-utils.js';
  * hash section of the URL.
  */
 
```

Comment:
For a lot of these utility functions we're passing in openmct, do we need them or can we just replace with the openmct (router usually) functions that they're then calling? Or is it to complicated with all the chaining?

Reply:
I agree, they should be moved into the router and `openmctLocation.js` retired. It's too confusing to developers having these functions split between the router and `openmctLocation.js`.